cmd/kpod/main.go

```diff
@@ -15,6 +15,7 @@ func main() {
 	if reexec.Init() {
 		return
 	}
+	logrus.SetLevel(logrus.ErrorLevel)
```

Comment:
It defaults to Info level, which ends up with us getting info messages in kpod commands.  Not something we want.